indigo_app/views/works.py

```diff
@@ -480,9 +633,9 @@ def get_context_data(self, **kwargs):
             })
         commencement = commencement + [{
             'rel': 'commenced',
-            'work': w,
-            'date': w.commencement_date,
-        } for w in self.work.commenced_works.all()]
+            'work': c.commenced_work,
```

Comment:
In addition, we need to include all the works that commenced this one, currently we just pull in the primary one.